notify/slack/slack.go

```diff
@@ -206,12 +213,36 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 	if err != nil {
 		return true, notify.RedactURL(err)
 	}
-	defer notify.Drain(resp)
+	defer resp.Body.Close()
 
 	// Only 5xx response codes are recoverable and 2xx codes are successful.
```

Comment:
the comment would need an update.

Reply:
Removed this one and added a comment clarifying that sometimes errors come with a 200 response code.